### PR TITLE
[x] Science! Continued

### PR DESCRIPTION
**Pull request by Flupster**
Taking over for Z-Key Aerospace due to inactivity

### CODE REVIEW

File: NetKAN/xScience.netkan [hunk 1]
```diff
@@ -1,27 +1,18 @@
 {
     "spec_version": 1,
     "identifier": "xScience",
-    "$kref": "#/ckan/spacedock/234",
+    "$kref": "#/ckan/github/Flupster/KSP-X-Science",
     "license": "CC-BY-NC-SA-4.0",
     "author": ["Z-Key Aerospace", "Brodrick"],
-    "abstract": "The Science Report and Checklist for KSP.  [x] Science! keeps track of the science experiments you have completed and recovered to Kerbal Space Center. It also tracks science you have gathered which is held on vehicles and kerbals.  No matter what game scene you are in, what you current vehicle is, [x] Science! always reports on all science held anywhere.",
-    "resources": {
+    "abstract": "The Science Report and Checklist for KSP.  [x] Science! keeps track of the science experiments you have completed and recovered to Kerbal Space Center. It also tracks science you have gathered which is $    "resources": {
```
Comment (DasSkelett) [inline NetKAN/xScience.netkan]: You deleted some text and a new line here by accident

Comment (Flupster) [reply NetKAN/xScience.netkan]: I'm stupid! fixed, thanks for noticing

**Comment by DasSkelett** [comment]
Do you want to add yourself as an author?

Also, the name will differ to the original -> `"name": "KSP X Science",` (updated) vs `"name": "[x] Science!",` (original).

Also, it will conflict with the odd usage of the 'v' in the version strings of the previous versions:
grafik
I think this can be resolved by removing it from the old .ckans?

**Comment by Flupster** [comment]
> Do you want to add yourself as an author?
> 
> Also, the name will differ to the original -> `"name": "KSP X Science",` (updated) vs `"name": "[x] Science!",` (original).
> Edit: This is because CKAN pulls the name from the GitHub repo name.
> If you add a explicit `"name": "[x] Science!",` to the netkan, it should override this.
> One more edit: I think this also is the reason, why CKAN can't read the .version file.
> 
> Third, it will conflict with the odd usage of the 'v' in the version strings of the previous versions, CKAN thinks the older versions are newer ones (if you have 1.4 marked ad compatible):
> grafik
> I think this can be resolved by removing it from the old .ckans?

I decided to just create from scratch, this means if the original developer comes back he can merge my PR and [x] Science! can live on, for now [x] Science! Continued (this) will just be here so people using ckan can grab the mod magically :)

**Comment by linuxgurugamer** [comment]
it will conflict with the odd usage of the 'v' in the version strings of the previous versions, CKAN thinks the older versions are newer ones (if you have 1.4 marked ad compatible):
> grafik
> I think this can be resolved by removing it from the old .ckans?

Would be better to do an epoch rather than removing the old ones

**Comment by DasSkelett** [comment]
@Flupster now decided to make it a separate mod, so this issue is resolved.

Also I think it now would make sense to create a new forum thread, @Flupster, so you can communicate with the users outside the comment section of the original thread.

**Comment by HebaruSan** [comment]
#rebuild

File: NetKAN/xScienceContinued.netkan [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "spec_version": 1,
+    "identifier": "xScienceContinued",
+    "name": "[x] Science! Continued",
+    "$kref": "#/ckan/github/Flupster/KSP-X-Science",
+    "license": "CC-BY-NC-SA-4.0",
+    "author": ["Z-Key Aerospace", "Brodrick", "Flupster"],
+    "abstract": "The Science Report and Checklist for KSP.  [x] Science! keeps track of the science experiments you have completed and recovered to Kerbal Space Center. It also tracks science you have gathered which is held on vehicles and kerbals.  No matter what game scene you are in, what you current vehicle is, [x] Science! always reports on all science held anywhere.",
+    "resources": {
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/107661-ksp-105-x-science",
+        "repository": "https://github.com/Flupster/KSP-X-Science"
```
Comment (Tekaoh) [inline NetKAN/xScienceContinued.netkan]: Should `repository:` be removed in order to rely on the conversion from $kref instead?
```suggestion
```

File: NetKAN/xScienceContinued.netkan [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "spec_version": 1,
+    "identifier": "xScienceContinued",
+    "name": "[x] Science! Continued",
+    "$kref": "#/ckan/github/Flupster/KSP-X-Science",
+    "license": "CC-BY-NC-SA-4.0",
+    "author": ["Z-Key Aerospace", "Brodrick", "Flupster"],
+    "abstract": "The Science Report and Checklist for KSP.  [x] Science! keeps track of the science experiments you have completed and recovered to Kerbal Space Center. It also tracks science you have gathered which is held on vehicles and kerbals.  No matter what game scene you are in, what you current vehicle is, [x] Science! always reports on all science held anywhere.",
+    "resources": {
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/107661-ksp-105-x-science",
```
Comment (Tekaoh) [inline NetKAN/xScienceContinued.netkan]: Suggestion: Wildcard URL.
Better suggestion: @Flupster Make a new forum thread for your continuation and link it instead.
```suggestion
        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/107661-*",
```

**Comment by HebaruSan** [comment]
Two mods suggest or recommend xScience. Should this mod have `"provides": [ "xScience" ],` to pull it in in those cases?

**Comment by Tekaoh** [comment]
> Two mods suggest or recommend xScience. Should this mod have `"provides": [ "xScience" ],` to pull it in in those cases?

One of those mods is mine. I'll likely just make a PR to suggest Flupster's continuation instead anyways.

**Comment by Tekaoh** [comment]
> Two mods suggest or recommend xScience. Should this mod have `"provides": [ "xScience" ],` to pull it in in those cases?

Alternately, if this PR is able to be merged, I can take a stab at adding `replaced_by:` to the old xScience for the sake of testing that out.

**Comment by DasSkelett** [comment]
Does a `replaced_by` relationship imply a `conflicts` (@HebaruSan / @politas) ?

**Comment by politas** [comment]
@DasSkelett No, any conflicts or provides would need to specified separately.

**Comment by Flupster** [comment]
Anything needed to be done by me before this gets merged?

**Comment by HebaruSan** [comment]
> Anything needed to be done by me before this gets merged?

Is there going to be a new forum thread? That was suggested above and sounds like a good idea.

**Comment by Flupster** [comment]
> > Anything needed to be done by me before this gets merged?
> 
> Is there going to be a new forum thread? That was suggested above and sounds like a good idea.

I don't see a point, there are no additional features or changes other than recompiling for newer versions of KSP, I'll continue to monitor the older thread but seeing as there are no additional features there's no need to make a new thread.

**Comment by linuxgurugamer** [comment]
@Flupster The problem with not creating a new thread is that you don't have control of the OP, which can cause issues down the road (I'm running into this with the OSE Workshop)

**Comment by Tekaoh** [comment]
@Flupster One example is that you would be able to update the version number in the thread title. It would also just make it clear that you're continuing it as another mod and no longer just updating the same mod.

**Comment by Flupster** [comment]
I've created a thread and changed the homepage in the netkan meta.

**Comment by Tekaoh** [comment]
Hm, I wonder if a new thread takes a few minutes before being available to anyone else? I was on my way over there to give you kudos but got a 404...

**Comment by Flupster** [comment]
I'm still below 5 posts, it's in the process of being approved 😅

**Comment by Tekaoh** [comment]
Oh hahaha that's hilarious

**Pull request merged**